src/StepParityCost.h

```diff
@@ -0,0 +1,82 @@
+#ifndef STEP_PARITY_COST_H
+#define STEP_PARITY_COST_H
+
+#include "GameConstantsAndTypes.h"
+#include "StepParityDatastructs.h"
+
+namespace StepParity
+{
+	const int DOUBLESTEP= 850;
+	const int BRACKETJACK= 20;
+	const int JACK= 30;
+	const int JUMP= 0;
+	const int SLOW_BRACKET=300;
+	const int TWISTED_FOOT=100000;
+	const int BRACKETTAP= 400;
+	const int HOLDSWITCH= 55;
+	const int MINE= 10000;
+	const int FOOTSWITCH= 325;
+	const int MISSED_FOOTSWITCH= 500;
+	const int FACING= 2;
+	const int DISTANCE= 6;
+	const int SPIN= 1000;
+	const int SIDESWITCH= 130;
+	const int CROWDED_BRACKET = 0;
+	const int OTHER = 0;
+
+	// 0.1 = 1/16th note at 150bpm. Jacks quicker than this are harder.
+	// Above this speed, footswitches should be prioritized
+	const float JACK_THRESHOLD = 0.1;
```

Comment:
Yeah, they have different uses. 
`JACK_THRESHOLD` is used to decide whether it would be easier to hit something as a jack or a footswitch.
And `JACK_CUTOFF` in TechCounts.cpp is used to decide whether a given jack is fast enough to be counted as tech.

Out of curiosity, I just tried replacing the value of `JACK_THRESHOLD` with `JACK_CUTOFF`, and it produces pretty wildly different results.